apps/L1G/restaurant/restaurant-frontend/src/utils/ProfilePicCloudinary.ts

```
@@ -0,0 +1,23 @@
+const UPLOAD_PRESET = process.env.NEXT_PUBLIC_CLOUDINARY_UPLOAD_PRESET!;
```

ESLint

[error] Filename is not in kebab case. Rename it to `profile-pic-cloudinary.ts`.

(unicorn/filename-case)